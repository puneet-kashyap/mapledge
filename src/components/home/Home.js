import React from 'react';
import { Button } from '@material-ui/core';
import './home.css';
import assets from '../../assets/constants';
import titleImg from '../../images/main-img.png';
class Home extends React.Component {
  render() {
    return (
      <main>
        <section className="staffing">
          <div className="img-block">
            <img className="title-img" src={titleImg} alt="background" />
          </div>
          <div className="marketing-block">
            <h1 className="page-title">
              Recruitment &amp; <br /> Staffing solutions
            </h1>
            <div>
              <Button variant="contained" color="primary">
                Learn more
              </Button>
            </div>
          </div>
        </section>
        <section className="find">
          {assets.marketing.map((item, index) => {
            return (
              <div className="marketing" key={index}>
                <h2 className="marketing-name">{item.name}</h2>
                <p>{item.desc}</p>
                <h4>{item.slogan}</h4>
                <Button variant="contained">Learn more</Button>
              </div>
            );
          })}
        </section>
      </main>
    );
  }
}

export default Home;
