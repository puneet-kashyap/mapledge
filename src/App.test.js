import React from 'react';
import Home from './components/home/Home';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';


it('Home component renders correctly', () => {
  const tree = renderer.create(<Router><Home/></Router>).toJSON();
  expect(tree).toMatchSnapshot();
})


