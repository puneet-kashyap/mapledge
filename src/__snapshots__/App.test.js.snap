// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders correctly 1`] = `
<main>
  <section
    className="staffing"
  >
    <div
      className="img-block"
    >
      <img
        alt="background"
        className="title-img"
        src="main-img.png"
      />
    </div>
    <div
      className="marketing-block"
    >
      <h1
        className="page-title"
      >
        Recruitment & 
        <br />
         Staffing solutions
      </h1>
      <div>
        <a
          className="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-contained-12 MuiButton-containedPrimary-13 MuiButton-raised-15 MuiButton-raisedPrimary-16"
          href="/about"
          onBlur={[Function]}
          onClick={[Function]}
          onContextMenu={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex="0"
        >
          <span
            className="MuiButton-label-2"
          >
            Learn more
          </span>
          <span
            className="MuiTouchRipple-root-30"
          />
        </a>
      </div>
    </div>
  </section>
  <section
    className="find"
  >
    <div
      className="marketing"
    >
      <h2
        className="marketing-name"
      >
        FIND WORK
      </h2>
      <p>
        If you are an Industrial Skilled Trade or General Worker - looking
            to advance your career or you are a Job Seeker - looking for
            flexible work schedules or you simply want to make extra money in
            addition to your main income
      </p>
      <h4>
        YOU CAME TO THE RIGHT PLACE.
      </h4>
      <a
        className="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-contained-12 MuiButton-raised-15"
        href="/contact"
        onBlur={[Function]}
        onClick={[Function]}
        onContextMenu={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex="0"
      >
        <span
          className="MuiButton-label-2"
        >
          Contact Us
        </span>
        <span
          className="MuiTouchRipple-root-30"
        />
      </a>
    </div>
    <div
      className="marketing"
    >
      <h2
        className="marketing-name"
      >
        FIND WORKER
      </h2>
      <p>
        If you are a Business looking for solutions to your Staffing
            needs, be it a Single Hire for a difficult to fill position or
            Staffing support of your Contingent Work Force....
      </p>
      <h4>
        YOU CAME TO THE RIGHT PLACE.
      </h4>
      <a
        className="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-contained-12 MuiButton-raised-15"
        href="/contact"
        onBlur={[Function]}
        onClick={[Function]}
        onContextMenu={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex="0"
      >
        <span
          className="MuiButton-label-2"
        >
          Contact Us
        </span>
        <span
          className="MuiTouchRipple-root-30"
        />
      </a>
    </div>
  </section>
</main>
`;
